### DISCUSSION TITLE
TuringCourses/major/computer_networks/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#37] [General]
# TuringCourses/major/computer_networks/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/major/computer_networks/

**Comment by CSWellesSun** [comment]
放上个人笔记：[ComputerNetwork](https://github.com/CSWellesSun/ZJUCourse/tree/main/%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%BD%91%E7%BB%9C)，里面包含高艺上课笔记和自顶而下的笔记。但是这门课上课、实验和考试有点貌合神离，最后考试还是建议刷王道，高艺班实验经过改革之后使用的是CS144的实验，我个人感觉前几个实验的测试点强度不高，到最后整合的时候容易暴露出前面的BUG，修起来难度有点大。

**Reply by NeilAlfred** [reply]
附议

**Comment by Unc1eW4ng** [comment]
这门课的上课体验极度令人不适，对于教改班而言，取缔了GNS3 相关的实验作为附加分，但是期末考中仍然会出现相关的试题，回想课程也只能从老师的一句“选做的实验也要去看”中找到相关的蛛丝马迹，故期末复习时还是建议和其他班的同学取一下考试范围的交集。同时课程大project切勿勉强，若发现选的题目涉及大量课程范围之外的内容，建议主动找老师替换，似乎有同学成功向老师提出自己想的一个题目作为替换。
